news/2018-02-25-project-loved-week-of-february-25th.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title: "Project Loved: Week of February 25th"
+date: 2018-02-25 18:00:00 +0000
+---
+
+The [Project Loved](https://osu.ppy.sh/community/forums/120) voting schedule is changing slightly starting this week! Come check out the new picks for all four modes now!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello everyone! Welcome to the last [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of February 2018! We are slightly changing up the voting format starting this week based on feedback from the community. This change will simplify the Loved voting schedule, as well as make it a lot easier to follow for those who care about more than just one gamemode!
```

Comment:
`[...] as well as make it [...]`
this reads weird to me, but I'm not sure on how to rephrase it :/